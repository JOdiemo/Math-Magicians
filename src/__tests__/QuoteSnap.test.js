import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

describe('Should Match Quote Page', () => {
  it('Identical to Quote Page', () => {
    const QouteSnap = renderer.create(<Quote />).toJSON;
    expect(QouteSnap).toMatchSnapshot();
  });
});
