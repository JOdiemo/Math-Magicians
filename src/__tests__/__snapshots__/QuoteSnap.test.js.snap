// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Match Quote Page Identical to Quote Page 1`] = `[Function]`;
